frappe/__init__.py

```diff
@@ -140,7 +140,11 @@ def set_user_lang(user, user_language=None):
 if typing.TYPE_CHECKING:
 	from frappe.database.mariadb.database import MariaDBDatabase
 	from frappe.database.postgres.database import PostgresDatabase
+	from pypika import Query
+
 	db: typing.Union[MariaDBDatabase, PostgresDatabase]
+	qb: Query
```

Comment:
Using the base class is okay here. We use Postgres and MySQL, both only override the base class methods. 👍